src/tokenserver/db/models.rs

```diff
@@ -211,6 +214,9 @@ impl TokenserverDb {
         "#;
         const DEFAULT_CAPACITY_RELEASE_RATE: f32 = 0.1;
 
+        let mut metrics = self.metrics.clone();
+        metrics.start_timer("tokenserver.storage.get_best_node", None);
```

Comment:
You may have intended this but I just wanted to note that Cadence hardcodes a prefix that we set to the value of `Settings::statsd_label` so this emits "syncstorage.tokenserver.storage.get_best_node".

We could overwrite tokenserver's `statsd_label` to use "tokenserver" instead or look to the future with "syncstorage.tokenserver" or maybe it makes more sense to keep it as you have it 🤷‍♂️

Reply:
Ooo I didn't notice this. I think it makes sense to overwrite `statsd_label` to `"syncstorage.tokenserver"` for Tokenserver. I'll make the change